_includes/templates/iot-gateway/request-converter-json-config.md

```diff
@@ -26,37 +32,49 @@ Mapping subsection will look like:
 
 ```json
     {
-      "url": "getdata",
-      "httpMethod": "GET",
-      "httpHeaders": {
-        "ACCEPT": "application/json"
+  "url": "getdata",
+  "httpMethod": "GET",
+  "httpHeaders": {
+    "ACCEPT": "application/json"
+  },
+  "allowRedirects": true,
+  "timeout": 0.5,
+  "scanPeriod": 5,
+  "converter": {
+    "type": "json",
+    "deviceNameJsonExpression": "SD8500",
+    "deviceTypeJsonExpression": "SD",
+    "attributes": [
+      {
+        "key": "serialNumber",
+        "type": "string",
+        "value": "${serial}"
+      }
+    ],
+    "telemetry": [
+      {
+        "key": "Maintainer",
+        "type": "string",
+        "value": "${Developer}"
       },
-      "allowRedirects": true,
-      "timeout": 0.5,
-      "scanPeriod": 5,
-      "converter": {
-        "type": "json",
-        "deviceNameJsonExpression": "SD8500",
-        "deviceTypeJsonExpression": "SD",
-        "attributes": [
-          {
-            "key": "serialNumber",
-            "type": "string",
-            "value": "${serial}"
-          }
-        ],
-        "telemetry": [
-          {
-            "key": "Maintainer",
-            "type": "string",
-            "value": "${Developer}"
-          }
-        ]
+      {
+        "key": "Vendor",
+        "type": "string",
+        "value": "${Client}",
+        "tsField": "${timestampField}",
+        "dayfirst": true
       }
-    }
+    ]
+  }
+}
 ```
 
-Also, you can combine values from MQTT message in attributes, telemetry and serverSideRpc section, for example:
+{% capture tsField %}
+**Note**: In **Mapping subsection** configuration section for `Vendor` timeseries key illustrates how to use the `tsField` and `dayfirst` options.
```

Comment:
Vendor usually is a producer of devices, and it can be saved in the attributes, because it cannot change. Please use a different key for demonstration, for example **pressure**